doc/user/content/sql/create-sink.md

```diff
@@ -183,21 +182,6 @@ FORMAT AVRO USING
     CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection;
 ```
 
-#### Get actual Kafka topic names
```

Comment:
Ah, fair enough. I removed them since the "actual" Kafka topic name is just the name that you typed in `CREATE SOURCE`. Amended to drop the word "actual" rather than the example entirely.

Reply:
:bow: